pkg/utils/writer.go

```diff
@@ -43,11 +43,17 @@ func (s *splitWriter) Write(b []byte) (int, error) {
 	for {
 		b = s.buffer.Bytes()
 		index := bytes.Index(b, []byte{'\n'})
-		if index < 0 {
+		if index > 0 {
```

Comment:
These aren't equivalent, right? I'm looking at this from my phone, so might be wrong (please disregard if so), but with the previous code if there's a `\n` at index `0` we handle it properly, but in the new code if we have a `\n` at index `0` we jump to the same branch as "no \n in this line".